RedHat-UX/red-hat-design-system
# docs(timestamp): update best practices

## What I did

1. Updated "Best practice" section for timestamp to show do/don't examples


## Testing Instructions

1. Check `<rh-timestamp>`'s ["Best practices" section](https://deploy-preview-2031--red-hat-design-system.netlify.app/elements/timestamp/guidelines/#best-practices) in the DP.

## Notes to Reviewers
